### PR TITLE
Replace the graph.yml with scaling at the runner

### PR DESCRIPTION
**Pull request by grdw**
This removes all the graph.yml overhead and replaces it
with a calculator which scales all the necessary graph
attributes.

### CODE REVIEW

**Comment by grdw** [comment]
@antw I assigned you as reviewer. Please feel to review this when time permits.

**Comment by antw** [comment]
@grdw Do you mind if we do this after the deploy? I *looove* the removal of graph.yml, but I'd rather not risk it so close to the next release.

**Comment by grdw** [comment]
> Do you mind if we do this after the deploy?

That's fine with me! 👍

File: lib/atlas.rb [hunk 1]
```diff
@@ -102,7 +99,6 @@
 
 require_relative 'atlas/runtime'
 require_relative 'atlas/exporter'
-require_relative 'atlas/exporter/essential_exporter'
 require_relative 'atlas/exporter/full_exporter'
```
[the remark this answers is not in the document]
Comment (grdw) [reply lib/atlas.rb]: Done!

*Note to self: this rename from FullExporter -> Exporter must also occur at ETEngine when bumping the Atlas gem.*

**Pull request merged**